docs/clustering/certificate-management.md

```diff
@@ -0,0 +1,51 @@
+# Certificate Management
+
+## Development
+
+Out of the box harperdb generates certificates that are used when HarperDB nodes are clustered together to securely share data between nodes. These certificates are meant for development/testing purposes. Because these certificates do not have CNs (Common Names) that will match the DNS/IP of the HarperDB node, the following settings (see the full [configuration file](../configuration.md) docs for more details) are defaulted & recommended for ease of development:
+
+```
+clustering:
+  tls:
+    certificate: ~/hdb/keys/certificate.pem
+    certificateAuthority: ~/hdb/keys/ca.pem
+    privateKey: ~/hdb/keys/privateKey.pem
+    insecure: true
+    verify: true
+```
+
+The certificates that HarperDB generates are stored in your `HDB_ROOT/keys/`.
+
+Insecure is set to `true` to accept the certificate CN mismatch due to development certificates.
+
+Verify is set to `true` to enable mutual TLS between the nodes.
+
+## Production
+
+In a production environment, we recommend using your own Certificate Authority, or a public Certificate Authority such as LetsEncrypt to generate certs for your HarperDB cluster. This will let you generate certificates with CNs that match the DNS of your nodes.
```

Comment:
```suggestion
In a production environment, we recommend using your own Certificate Authority, or a public Certificate Authority such as LetsEncrypt to generate certs for your HarperDB cluster. This will let you generate certificates with CNs that match the FQDN of your nodes.
```